modules/p2/ding_interaction/plugins/content_types/interaction_pane.inc

```diff
@@ -42,7 +42,19 @@ function ding_interaction_content_type_render($subtype, $conf, $panel_args, $con
   ctools_include('ajax');
   // @TODO Change $output to render array and use #attached for files.
   drupal_add_js(drupal_get_path('module', 'ding_interaction') . '/js/ding_interaction.js');
-  $data = ding_interaction_content(NULL, $conf['count']);
+  drupal_add_css(drupal_get_path('module', 'ding_interaction') . '/css/ding_interaction.css');
+
+  $extra = $module_list = array();
+
+  // Force select an interaction depending on get parameters.
+  if (!empty($_GET['interaction']) && user_access('administer personalisation')) {
+    $module_list[$_GET['interaction']] = module_invoke($_GET['interaction'], 'ding_interaction_info');
+    if (!empty($_GET['interaction_extra'])) {
+      $extra = $_GET['interaction_extra'];
```

Comment:
The drupal_get_query_parameters is usually used when passing the queries to another link.
When requesting a specific parameter the $_GET array is used directly.